packages/perseus/src/widgets/radio/multiple-choice-component.new.tsx

```diff
@@ -126,69 +121,17 @@ const BaseRadio = ({
         }
 
         prevReviewModeRubric.current = reviewModeRubric;
-    }, [apiOptions, choices, isLastUsedWidget, reviewModeRubric]);
+    }, [apiOptions, isLastUsedWidget, reviewModeRubric, choices]);
 
-    // When a particular choice's `onChange` handler is called, indicating a
-    // change in a single choice's values, we need to call our `onChange`
-    // handler in order to notify our parent. However, our API with our parent
-    // is that we always provide *all* values for *all* choices, even if just
-    // one choice's values changed. (This is because sometimes an interaction
-    // with one choice can affect many choices, like how checking a new answer
-    // will usually cause the old answer to become unchecked.)
-    //
-    // So, given the new values for a particular choice, compute the new values
-    // for all choices, and pass them to `onChange`.
-    //
-    // `newValues` is an object with a `checked` key. It contains a boolean value
-    // specifying the new checked value of this choice.
-    function updateChoice(
-        choiceIndex: number,
-        newValues: Readonly<{
-            checked: boolean;
-        }>,
-    ): void {
-        // Get the baseline `checked` values. If we're checking a new answer
-        // and multiple-select is not on, we should clear all choices to be
-        // unchecked. Otherwise, we should copy the old checked values.
-        let newCheckedList;
-        if (newValues.checked && !multipleSelect) {
-            newCheckedList = choices.map((_) => false);
-        } else {
-            newCheckedList = choices.map((c) => c.checked);
-        }
-
-        // Update this choice's `checked` values.
-        newCheckedList[choiceIndex] = newValues.checked;
-
-        onChange({
-            checked: newCheckedList,
-        });
-    }
-
-    // some commonly used shorthands
-    const isMobile = apiOptions.isMobile;
-
-    const firstChoiceHighlighted = choices[0].highlighted;
-    const lastChoiceHighlighted = choices[choices.length - 1].highlighted;
-
-    const className = classNames(
+    const choiceListClassName = classNames(
         "perseus-widget-radio",
-        !editMode && "perseus-rendered-radio",
-        css(
-            styles.radio,
-            styles.responsiveRadioContainer,
```

Comment:
I consolidated radio, responsiveRadioContainer, and fieldSet from the aphrodite styles into a single class as this was quite silly.